src/components/AppHeader/MobileMenu/SubmenuContent.tsx

```diff
@@ -6,146 +9,65 @@ type TSubmenuContent = {
     renderContentFor: TSubmenuSection;
 };
 
+type TMenuItem = {
+    Icon: IconTypes;
+    fontWeight: ComponentProps<typeof Text>['weight'];
+    label: string;
+};
+
+const MenuItem = ({ Icon, fontWeight, label }: TMenuItem) => {
+    const { isMobile } = useDevice();
+
+    return (
+        <>
+            <Icon iconSize='xs' />
+            <Text className='ml-[1.6rem]' size={isMobile ? 'md' : 'sm'} weight={fontWeight}>
```

Comment:
nit: I think we could have passed the text sizing down from the parent since the value is the same below